src/client/service-components/EditableTextArea.js

```diff
@@ -0,0 +1,91 @@
+import React, {useState, useContext} from 'react';
+import {UserSettingsContext} from '../context-store/user-settings-context.js';
+import EditIcon from '../images/edit-icon-18.svg';
+
+import './EditableTextArea.css';
+
+
+export const EditableTextArea = (props) => {
```

Comment:
Why do you have different components with the same logic for input and text area?

Reply:
I'm working on it

Reply:
So, are you going to make refactoring for these components in this PR?

Reply:
Yes, I'm trying to finish it today.